__tests__/integration/config-invalid/expected-output.txt

```diff
@@ -0,0 +1,2 @@
+⠼ snowpack installing...
+✖ package.json: snowpack.webDependencies is not of a type(s) array
```

Comment:
This is an example of a friendly error message users can see when their config is wrong. We could probably get more sophisticated over time, but this isn’t a bad start.